tests/e2e/env/utils/index.js

```diff
@@ -1,7 +1,8 @@
-const getAppRoot = require( './app-root' );
+const { getAppRoot, getAppName } = require( './app-root' );
 const getTestConfig = require( './test-config' );
 
 module.exports = {
     getAppRoot,
+	getAppName,
```

Comment:
This line is indented using tabs and the others are using spaces. I'm not sure which one is the correct option.

Reply:
> Could you please clarify?

@rodrigoprimo I had to update package-lock.json to get Travis to run. Indentation fixed in `7c509d8`.